src-ui.v3/src/Main.hs

```diff
@@ -136,6 +138,27 @@ bodyElement4 = do
       display (V.length <$> dynPackages0)
       text ")"
 
+    -- search box
+    searchInputE <- elAttr "div" ("class" =: "item search right clearfix") $ do
+              divClass "text" $ text "Package Search"
+              sVal0 <- inputElement $ def & inputElementConfig_elementConfig . elementConfig_initialAttributes .~ fold
+                ["class" =: "input-search","placeholder" =: "search..."]
+              debounce 0.1 $ _inputElement_input sVal0
```

Comment:
The UI felt a little bit better when I bumped this to 0.2 rather than 0.1, but see how it is for you because my perf testing was all over the shop. Haha

Reply:
I think we can comeback to this after we put style in the code.